src/plugins/VMProfileLinuxSupportPlugin/VMProfileLinuxSupportPlugin.c

```diff
@@ -171,7 +177,7 @@ primitiveDLSymInLibrary(void)
 	symName = malloc(sz + 1);
 	strncpy(symName, firstIndexableField(nameObj), sz);
 	symName[sz] = 0;
-	lib = dlopen(libName, RTLD_LAZY | RTLD_NODELETE);
```

Comment:
Hi bencoman.

RTLD_NODELETE doesn't exist in OpenBSD: http://man.openbsd.org/dlopen.3